lib/web3/requestmanager.js

```diff
@@ -161,9 +159,8 @@ RequestManager.prototype.setProvider = function (p) {
  * @todo cleanup number of params
  */
 RequestManager.prototype.startPolling = function (data, pollId, callback, uninstall) {
-    this.polls['poll_'+ pollId] = {data: data, id: pollId, callback: callback, uninstall: uninstall};
+    this.polls[pollId] = {data: data, id: pollId, callback: callback, uninstall: uninstall};
```

Comment:
the point of this is to prevent keys starting with numbers, as some browser have problems with that.